rules/crawlers-user-agents.data

```diff
@@ -36,3 +36,532 @@ Owlin bot
+Go-Ahead-Got-It
+Gotit
+GoZilla
+Go!Zilla
+Grabber
+GrabNet
+Grafula
+GrapeFX
+GrapeshotCrawler
+GridBot
+GT::WWW
+Haansoft
+HaosouSpider
+Harvest
+Havij
+HEADMasterSEO
+Heritrix
+Hloader
+HMView
+HTMLparser
+HTTrack
+Humanlinks
+HybridBot
+Iblog
+IDBot
+Id-search
+IlseBot
+Image Fetch
+Image Sucker
+IndeedBot
+Indy Library
+InfoNaviRobot
+InfoTekies
+instabid
+Intelliseek
+InterGET
+Internet Ninja
+InternetSeer
+internetVista monitor
+ips-agent
+Iria
+IRLbot
+Iskanie
+IstellaBot
+JamesBOT
+Jbrofuzz
+JennyBot
+JetCar
+JikeSpider
+JOC Web Spider
+Joomla
+JustView
+Jyxobot
+Kenjin Spider
+Keyword Density
+Kozmosbot
+Lanshanbot
+Larbin
+LeechFTP
+LeechGet
+LexiBot
+Lftp
+LibWeb
+Libwhisker
+Lightspeedsystems
+Likse
+Linguee
+Linkdexbot
+LinkextractorPro
+LinkpadBot
+LinkScan
+LinksManager
+LinkWalker
+LinqiaMetadataDownloaderBot
+LinqiaRSSBot
+LinqiaScrapeBot
+Lipperhey
+Litemage_walker
+Lmspider
+LNSpiderguy
+Ltx71
+Magnet
+Mag-Net
+magpie-crawler
+Mail.RU_Bot
+Majestic12
+MarkMonitor
+MarkWatch
+Masscan
+Mass Downloader
+Mata Hari
+MauiBot
+Meanpathbot
+Mediatoolkitbot
+mediawords
+MegaIndex.ru
+Metauri
+MFC_Tear_Sample
+Microsoft Data Access
+Microsoft URL Control
+MIDown tool
+MIIxpc
+Mister PiX
+MJ12bot
+Mojeek
+Morfeus Fucking Scanner
+Mr.4x3
+MSFrontPage
+MSIECrawler
+Msrabot
+MS Web Services Client Protocol
+muhstik-scan
+Musobot
+Name Intelligence
+Nameprotect
+Navroad
+NearSite
+Needle
+Nessus
+NetAnts
+Netcraft
+netEstate NE Crawler
+NetLyzer
+NetMechanic
+NetSpider
+Nettrack
+Net Vampire
+Netvibes
+NetZIP
+NextGenSearchBot
+Nibbler
+NICErsPRO
+Niki-bot
+Nikto
+NimbleCrawler
+Ninja
+Nmap
+NPbot
+Nutch
+oBot
+Octopus
+Offline Explorer
+Offline Navigator
+Openfind
+OpenLinkProfiler
+Openvas
+OpenVAS
+OrangeBot
+OrangeSpider
+OutclicksBot
+OutfoxBot
+PageAnalyzer
+Page Analyzer
+PageGrabber
+page scorer
+PageScorer
+Panscient
+Papa Foto
+Pavuk
+pcBrowser
+PeoplePal
+PHPCrawl
+Picscout
+Picsearch
+PictureFinder
+Pimonster
+Pi-Monster
+Pixray
+PleaseCrawl
+plumanalytics
+Pockey
+POE-Component-Client-HTTP
+Probethenet
+ProPowerBot
+ProWebWalker
+Psbot
+Pump
+PxBroker
+QueryN Metasearch
+Quick-Crawler
+RankActive
+RankActiveLinkBot
+RankFlex
+RankingBot
+RankingBot2
+Rankivabot
+RankurBot
+RealDownload
+Reaper
+RebelMouse
+Recorder
+RedesScrapy
+ReGet
+RepoMonkey
+Ripper
+RocketCrawler
+Rogerbot
+s1z.ru
+SalesIntelligent
+SBIder
+ScanAlert
+Scanbot
+scan.lol
+Scrapy
+Screaming
+ScreenerBot
+Searchestate
+SearchmetricsBot
+Semrush
+SemrushBot
+SEOkicks
+SEOlyticsCrawler
+Seomoz
+SEOprofiler
+seoscanners
+SeoSiteCheckup
+SEOstats
+sexsearcher
+Shodan
+Siphon
+SISTRIX
+Sitebeam
+SiteExplorer
+Siteimprove
+SiteLockSpider
+SiteSnagger
+SiteSucker
+Site Sucker
+Sitevigil
+Slackbot-LinkExpanding
+SlySearch
+SmartDownload
+SMTBot
+Snake
+Snapbot
+Snoopy
+SocialRankIOBot
+sogouspider
+Sogou web spider
+Sosospider
+Sottopop
+SpaceBison
+Spammen
+SpankBot
+Spanner
+Spbot
+Spinn3r
+SputnikBot
+Sqlmap
+Sqlworm
+Sqworm
+Steeler
+Stripper
+Sucker
+Sucuri
```

Comment:
Does this refer to "Sucuri Integrity Monitor"?

Reply:
This one is an interesting one. 
If you are using Modsecurity. I doubt you too would be using Sucuri.
Nothing stops a bot from defining Sucuri as its user agent.

Thoughts anyone?